### DISCUSSION TITLE
Thank You

### DISCUSSION THREAD

**Discussion by leumas1960** [RS485/LogisticsPipes#1603] [General]
Awhile back, both fuzzy upgrades/functionality, and the power transportation upgrades were completely non-functional. Now, they work, and using the mod is way more fun, not to mention less of a nightmare. I just want to say thank you to whoever put the work into fixing that. The improved sat pipe functionality is sweet too. Now I can use many types of wood, and autocraft 14,000,000,000... types of tin at the same time.

**Comment by bziemons** [comment]
I am glad to hear that :)
